src/OmnichannelChatSDK.ts

```diff
@@ -902,6 +902,8 @@ class OmnichannelChatSDK {
 
                 if (isClientIdNotFoundErrorMessage(error)) {
                     exceptionThrowers.throwAuthContactIdNotFoundFailure(error, this.scenarioMarker, TelemetryEvent.GetChatToken, telemetryData);
+                } else if (isInvalidAuthentication(error)) {
```

Comment:
The `error` should be coming from `axios`. Can we add an additional check for that? 

```
if ((error as any)?.isAxiosError && isInvalidAuthentication(error))
```

Reply:
> if ((error as any)?.isAxiosError && isInvalidAuthentication(error))

Can adding isAxiosError condition

> if ((error as any)?.isAxiosError && isInvalidAuthentication(error))

Is there a way to validate before adding this condition, may be runtime use some fake axios call

Reply:
Added axios check.

There should not be a case where we get both InvalidAuthenticationError and not an axios error. That is because axios error is defined as specifically relating to axios call responses, and InvalidAuthentication is checking chatSDK's getChatToken response which relies on OCSDK which relies on axios to make the HTTP calls. If the HTTP calls fails with 401, it should be an axios error.